package.json

```diff
@@ -25,9 +25,9 @@
     "env-paths": "^2.2.0",
     "glob": "^7.1.4",
     "graceful-fs": "^4.2.3",
+    "make-fetch-happen": "^8.0.9",
```

Comment:
I guess the fetch requirement if pretty basic. Should we just use native fetch function. No need to again upgrade in couple of years :)

Reply:
There is (at the time of this writing) no native fetch function in Node.js.

Reply:
make-fetch-happen also deals with proxies which is a big thing for corporate users.

Reply:
ok proxy support would be critical requirement. No point implementing that.